CYLGame/www/templates/grid.html

```diff
@@ -404,7 +404,9 @@ <h3>Play the Game</h3>
       <p>This is where the user can play the game to test out different ideas.</p>
       <div id="play_container" style="margin-left: 1em;margin-right: 1em;">
       </div>
+      <button type="button" id="playAgain" disabled>Click to play again</button>
   </div>
+<!--  this is where the button comes in -->
```

Comment:
I'm not sure what you mean by this line.

Reply:
I will remove the comment. How do I merge? Thank you Chandler.